test/integration/targets/tests/tasks/main.yml

```
@@ -0,0 +1,31 @@
+- name: Assert truthy tests work
+  assert:
+    that:
+      - '"string" is truthy'
+      - '"" is not truthy'
+      - True is truthy
+      - False is not truthy
+      - true is truthy
+      - false is not truthy
+      - 1 is truthy
+      - 0 is not truthy
+      - '[""] is truthy'
+      - '[] is not truthy'
+      - '"on" is truthy(convert_bool=True)'
+      - '"off" is not truthy(convert_bool=True)'
```

Add some tests for dicts.
```suggestion
      - '"off" is not truthy(convert_bool=True)'
      - '{} is not truthy'
      - '{"key": "value"} is truthy'      
```